README.md

```diff
@@ -16,50 +16,110 @@ When you are done:
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
 # copy & paste your solution here
+def return_max(l):
+  if len(l)==0: 
+    return None
+  else: 
+      l.sort()
+  return l[-1]
```

Comment:
This is super clever! Good job.

It's worth nothing that the `sort` method has [variable time complexity](https://en.wikipedia.org/wiki/Timsort), so it might be better to just loop over the list for the sake of time.